gcs_oauth2_boto_plugin/test_oauth2_client.py

```diff
@@ -174,6 +174,34 @@ def _RunGetAccessTokenTest(self, expected_rapt=None):
         token_3.expiry)
     self.assertEqual(token_3.rapt_token, expected_rapt)
 
+class TestCreateP12ServiceAccountCredentials(unittest.TestCase):
+  
+  @mock.patch.object(oauth2_client.PKCS12Signer, 'from_string')
```

Comment:
This patcher is provided by the unittest mock which is available in py 3.3+. For the earlier versions,  It relies on mock third party lib. Will this compile for the earlier versions? https://github.com/GoogleCloudPlatform/gcs-oauth2-boto-plugin/blob/master/gcs_oauth2_boto_plugin/test_oauth2_client.py#L33

Reply:
It looks like before python version 3.3 we're importing mock in our tests, and it's same mock which is part of unit-test in python version > 3.3

```
mock is now part of the Python standard library, available as [unittest.mock](https://docs.python.org/dev/library/unittest.mock.html) in Python 3.3 onwards.
```

Reply:
Got it! thanks.